### ISSUE TITLE
Site has been unlisted until we decide what to do with this.

### ISSUE THREAD

**Issue by ChristopherTrimboli** [exokitxr/exokit-site-old#3]
One day when we have time I would like to perhaps make this a super interactive web app where people can join Multiplayer servers and do stuff.. most of the design work was pretty decent.

**Comment by avaer** [comment]
Yeah, nothing wrong with the site. Just temporarily wanted to reduce confusion while we figure out the design.
